### PR TITLE
hotfix: removed duplicated snackbar

### PR DESCRIPTION
**Pull request by giteshk11**
# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[x] Bug Fix
[ ] Refactor
[ ] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** 
- What was the root cause? 
   There were two snackbars one in App.vue and one in Home.vue 
- How do your changes fix the bug effectively?
   Removed the Snackbar from the Home.vue

# How to Test/Reproduce
Provide steps on how one can test your changes here. e.g.
- Open the app in your browser and you should see the snack bar
- Click on agree to make it disappear.

# Screenshots/casts
Screen Shot 2020-08-30 at 13 48 18

### CODE REVIEW

**Comment by joeelia** [comment]
This is only removal of code. Code looks good for Privacy SnackBar hotfix. When this PR gets pushed to staging it will help with the iOS / Android build not incur errors.

File: src/App.vue [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default {
     setSnackBarCookie() {
       this.showPrivacySnackBar = false;
       //set cookie with name 'snackbar'. Set value to 1 which means true. Set expiration to 7 days.
-      cookie.setCookie('privacy-snackbar', 1, 7);
+      cookie.setCookie('privacy-snackbar', 1, 365);
```
Comment (joeelia) [inline src/App.vue:77]: Good job changing this from 7 day to 365 day expiry

**Review by paulchrisluke — APPROVED** [review]
This looks great thank you for the contribution!

**Comment by paulchrisluke** [comment]
We may want to add a little padding to the bottom so it's not stuck to the bottom of the screen but besides that it looks good to merge

**Comment by giteshk11** [comment]
I can see padding is already there 
image

**Pull request merged**